src/popup/router/pages/Networks.vue

```diff
@@ -197,8 +219,36 @@ export default {
   }
 }
 
-.edit-btn {
-  margin-left: 5px;
-  margin-right: 0;
+.network {
+  .edit-btn {
+    margin-left: 5px;
+    margin-right: 0;
+  }
+
+  .expand {
+    font-size: 14px;
+    cursor: pointer;
```

Comment:
it is not necessary if .expland is a button

Reply:
I am not sure what is the issue here. 
<img width="334" alt="Screenshot 2020-10-09 at 17 26 38" src="https://user-images.githubusercontent.com/7085525/95595328-b67b5a00-0a54-11eb-9115-cf56c3e176e6.png">
This is the section, which the user is expanding and collapsing.

Reply:
`cursor: pointer` is the default behavior of `<button>`